### DISCUSSION TITLE
onSlideEnd not fired when using keyboard

### DISCUSSION THREAD

**Discussion by lisardo2** [andreruffert/rangeslider.js#362] [Q&A] [answered]
focus and controlling with arrow keys works. The onSlide callback function is fired every time the arrow key ist used. But the onSlideEnd is not fired when an arrow key is used - only when working with the mouse. But onSlide is not really useful when working with a mouse - he fires hundreds of times when the handle is moved. 

Is there a possibility to bind some action on onSlideEnd for mouse and arrow keys? 

Thanks!

**Comment by andreruffert** [comment, marked as answer]
I'd suggest using the `input` or `change` event instead of the callback functions. e.g.

```js
const $element = $('input[type="range"]');

$element
  .rangeslider({
    polyfill: false
  })
  .on('input', function(e) {
    // do your thing...
  });
```
